### PR TITLE
update doc for android MainApplication.java

### PR DESCRIPTION
**Pull request by wasabia**

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @lacker and @satya164 to be potential reviewers.

**Comment by facebook-github-bot-1** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -295,7 +295,7 @@ var featured = [
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.bdsdiet_app',
     infoLink: 'https://www.crunchbase.com/organization/bds-diet#/entity',
     infoTitle: 'Bdsdiet provides real estate brokerage services through web and live agents in Korea.',
-  },
+  }
```
Comment (satya164) [inline website/src/react-native/showcase.js]: This change is unrelated

**Comment by satya164** [comment]
Please sign the CLA so we can accept the PR.

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by wasabia** [comment]
@satya164 done

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1180923375303332/int_phab) to review internal test results.
